library/src/main/java/com/xwray/groupie/GroupAdapter.java

```diff
@@ -209,6 +209,15 @@ public void remove(@NonNull Group group) {
         groups.remove(position);
         notifyItemRangeRemoved(itemCountBeforeGroup, group.getItemCount());
     }
+    
+    public void removeAt(int index) {
+        Group group = getGroup(index);
```

Comment:
Though that's one way, I ran into an edge case where I needed this method, though there is remove(Group group), have to check my code and see, I think I needed an action where I don't know the group's but I know only their positions and where they should be added or removed dynamically based on certain criteria.